mistletoe/html_renderer.py

```diff
@@ -193,12 +199,20 @@ def render_document(self, token: block_token.Document) -> str:
         inner = '\n'.join([self.render(child) for child in token.children])
         return '{}\n'.format(inner) if inner else ''
 
-    @staticmethod
-    def escape_html(raw: str) -> str:
+    def escape_html_text(self, s: str) -> str:
         """
-        This method is deprecated. Use `html.escape` instead.
+        Like `html.escape()`, but this  looks into the current rendering options
+        to decide which of the quotes (double, single, or both) to escape.
+
+        Intended for escaping text content. To escape content of an attribute,
+        simply call `html.escape()`.
         """
-        return html.escape(raw)
+        s = html.escape(s, quote=False)
```

Comment:
As `html.escape` doesn't seem to be a native function, in-lining it here should be faster?

Reply:
An advantage of inlining it would be to avoid the clever but scary patching of `html._charref` done in `span_tokenizer.tokenize()`.

Reply:
`html._charref` is used for `html.unescape` though? :)

Reply:
Yes, you're right. It won't help for that.

Reply:
FYI, I in-lined the code in the end. Not that it would be visibly faster, but I guess it's the way to go in this case.